app/assets/stylesheets/home.scss

```diff
@@ -1,6 +1,6 @@
 .masthead-video {
   background-color: black;
-  position: fixed;
+  position: absolute;
   top: 50%;
```

Comment:
Since it's being introduced in this PR, I'd say let's put it back to `fixed`, deploy, and then create an issue to properly tackle the the problem.